addons/website/static/src/snippets/s_popup/001.scss

```diff
@@ -64,3 +71,11 @@
         }
     }
 }
+
+#oe_main_menu_navbar:not(.o_hidden) + #wrapwrap main {
```

Comment:
Why do you check if not 'o_hidden' on '#oe_main_menu_navbar' to know if the navbar is displayed or not ?
Because I dont see that this class (o_hidden) is added on the navbar when not connected.. For me the navbar is simply removed...

Reply:
![Screenshot-from-2021-01-11-16-59-41](https://user-images.githubusercontent.com/25147715/104177271-831ee180-542e-11eb-9870-83ea77a8c87e.png)
When you are in edit mode, the navbar is hidden so if I don't add `not(.o_hidden)`, it will have space at the top when the popup is open